api/database/factories/ModelFactory.php

```diff
@@ -106,3 +107,20 @@
 
     return compact('token') + $body;
 });
+
+$factory->define(App\Models\ArticlePermalink::class, function ($faker) {
+    return [
+        'permalink' => str_random(255),
```

Comment:
Hmm I got errors from calling `$faker->slug` directly, as there was issues with the uniqueness of the generated value.